org.civicrm.styleguide/tests/protractor/lib/navbar.js

```diff
@@ -0,0 +1,48 @@
+var prtr = require('./protractor-util');
+
+/**
+ * The CiviCRM navbar waits for the mouse to hover before exposing nested elements.
+ * This constant specifies the #milliseconds to hover an element before moving
+ * on to the next child.
+ *
+ * This is likely to be a systemic performance drag on the test-suite...
+ *
+ * @type {number}
+ */
+var HOVER_DLEAY = 250;
```

Comment:
@totten nitpick: typo in allcaps, these things can develop a life of their own :D